cmd/cstorvolumeclaim/controller_base.go

```diff
@@ -242,7 +242,7 @@ func (c *CVCController) deleteCVC(obj interface{}) {
 // as syncing informer caches and starting workers. It will block until stopCh
 // is closed, at which point it will shutdown the workqueue and wait for
 // workers to finish processing their current work items.
-func (c *CVCController) Run(threadiness int, stopCh <-chan struct{}) error {
+func (c *CVCController) Run(threadiness int, stopCh <-chan struct{}) {
```

Comment:
why this change required?

Reply:
golang-ci was complaining about the error not handled